package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "prepare": "node lib/util/prepare-swagger-ui",
     "prepublishOnly": "npm run prepare",
-    "test": "standard && tap --100 test/**/*.js && npm run typescript",
-    "test:ci": "standard && tap test/**/*.js --coverage-report=lcovonly && npm run typescript",
+    "test": "standard && tap --100 test/*.js test/**/*.js test/**/**/*.js && npm run typescript",
```

Comment:
I think this change is not needed as `test/**/*.js` means any files under `test` folder.

Reply:
I used to think that too but instead running with the npm it does not consider deeper level files...
As a notice: in fact the coverage shown in the repository badge is currently 46% instead of 100%...

Reply:
I face the same issue when I do a clean install of packages. It seems like `tap` changes the behavior of glob.

Reply:
Yep... Apparently it is a bug on their side..

Reply:
Yeah, probably a bug on their side

Reply:
@giovanni-bertoncelli
I finally see the cause, it should be the problem with bash, npm or yarn.
Can you test if adding a quote help to resolve it? It is a little bit too long if we include all the nested folder case.
```
"test": "standard && tap --100 'test/**/*.js' && npm run typescript",
"test:ci": "standard && tap 'test/**/*.js' --coverage-report=lcovonly && npm run typescript",
```

Reply:
Should be fine now